src/components/dashboard/Chat/ChatController.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import ChatMessages from './Chat';
+import { socket } from '../../../actions/chat';
+
+class ChatController extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      input: '',
+      messages: []
+    }
+  }
```

Comment:
@bonham000 I'm interested to chat about this at some point.. ever since the Rem tutorials, I've just been simply writing:
```js 
this.state = {
    state: 'here'
}
``` 
and haven't been bothering with the constructor or the specific `this` reference. And at least in CRA environment, this seems to be fine. Is there a discernible difference do you know?